GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -785,25 +803,33 @@ Profile
     INPUT_RESOURCE
     {
       name = Ammonia
-      rate = 0.00695                    // 37530 units required for crop
+      rate = 0.00229                    // 15 units required per unit of crop, i.e. 200 days * 900 "4 seconds" per hour * 24 hours * rate == 15 * crop_size
     }
 
+    // Photosynthesis reaction: 2xH20 + CO2 -> O2 + CH20 + H20
+    // Note that CH20 is an intermediate carbohydrate, energy that goes into the food, in the actual food it can be any CxHyOz compound
     INPUT_RESOURCE
```

Comment:
Is H20 a Hydrogen molecule consisting of 20 bonded Hydrogen atoms or did you mean water H2O?

Reply:
That's a typo, it should be H2O

Reply:
Thought so 😉